src/stability_sdk/animation.py

```diff
@@ -80,33 +76,27 @@ class AnimationSettings(param.Parameterized):
     locked_seed = param.Boolean(default=False)
 
 
-# TO DO: ability to specify backfill/interpolation method for each parameter
-# TO DO: ability to provide a function that returns a parameter value given some frame index
-# TO DO: inherit from param.String to add validation to these things
-# TO DO: should defaults be noop or opinionated? Maybe a separate object for opinionated defaults?
-class KeyframedSettings(param.Parameterized):
+class CameraSettings(param.Parameterized):
     """
     See disco/deforum keyframing syntax, originally developed by Chigozie Nri
     General syntax: "<frameId>:(<valueAtFrame>), f2:(v2),f3:(v3)...." 
     Values between intermediate keyframes will be linearly interpolated by default to produce smooth transitions.
     For abrupt transitions, specify values at adjacent keyframes.
     """
-    angle = param.String(default="0:(1)")
-    zoom = param.String(default="0:(1)")
+    angle = param.String(default="0:(0)", doc="Camera rotation angle in degrees for 2D mode")
+    zoom = param.String(default="0:(1)", doc="Camera zoom for 2D mode (<1 zooms out, >1 zooms in)")
```

Comment:
For clarity (and to address a common question/misunderstanding), it may be good to use the word "factor" here:
`Camera zoom factor for 2D mode (<1 zooms out, >1 zooms in)`

Reply:
Sounds good!